assets/javascripts/openqa.js

```diff
@@ -52,6 +52,13 @@ function addFlash(status, text, container) {
   return div;
 }
 
+function clearFlash() {
+  const flashMessages = $('#flash-messages').find('.alert.alert-primary.alert-dismissible');
+  flashMessages.each(function () {
+    $(this).remove();
+  });
```

Comment:
This can be achieved in a single line as jQuery always operates on lists:

```suggestion
  $('#flash-messages .alert.alert-primary.alert-dismissible').remove();
```

The design of jQuery is so unintuitive (and thus probably bad) that one can even easily miss cases where the design is useful.

**This is also easily doable without jQuery and I'd avoid using it in new code. So I suggest to use the following:**

```suggestion
  document.querySelectorAll('#flash-messages .alert.alert-primary.alert-dismissible').forEach(e => e.remove());
```

This code uses proper function names instead of "magic" ones and is thus much more self-explaining.